### PR TITLE
[50955] Rename time fields in the UI

### PR DESCRIPTION
**Pull request by cbliard**
https://community.openproject.org/wp/50955

Estimated time => Work
Remaining time => Remaining work
Progress (%) => % Complete

### CODE REVIEW

**Comment by oliverguenther** [comment]
This is scoped for 13.2., correct? So to merge this, we'll need to create a release branch first.

**Comment by cbliard** [comment]
Yes it is. I'll mark it as draft.

**Comment by cbliard** [comment]
But sadly draft PR cannot be reviewed.

**Comment by cbliard** [comment]
It's only a flaky test which is failing.

File: spec/features/work_packages/select/select_query_spec.rb [hunk 1]
```diff
@@ -83,7 +83,7 @@
     it 'shows the saved filters', js: true do
       filters.open
       filters.expect_filter_by 'Assignee', 'is (OR)', ['me']
-      filters.expect_filter_by 'Progress (%)', '>=', ['10'], 'percentageDone'
+      filters.expect_filter_by 'Percent Complete', '>=', ['10'], 'percentageDone'
```
Comment (ulferts) [inline spec/features/work_packages/select/select_query_spec.rb:86]: I believe this should be '% Complete'

**Review by ulferts — CHANGES_REQUESTED** [review]
Only one teeny tiny thing, @cbliard. Feel free to merge yourself after addressing.

File: spec/features/work_packages/select/select_query_spec.rb (hunk 1 above)
Comment (cbliard) [reply spec/features/work_packages/select/select_query_spec.rb:86]: Actually, it does not really matter. In `expect_filter_by`, the name is not used if there is a selector. I was surprised that it was still passing after renaming, so I changed it to something which is not meaningful (does not match selector or translated name) to make it a bit more self-explanatory that this parameter is actually not relevant.

**Pull request merged**